types/setting.go

```diff
@@ -729,6 +730,7 @@ var (
 		Category: SettingCategoryDangerZone,
 		Required: false,
 		ReadOnly: false,
+		Default:  DefaultPriorityClassName,
```

Comment:
No need to have a default value here, because this value will be controlled by chart values as we discussed.